package.json

```diff
@@ -19,7 +20,7 @@
     "prettier-watch:win32": "onchange \"src/**/*.ts\" -- prettier --write {{changed}}",
     "prettier-watch:darwin:linux": "onchange 'src/**/*.ts' -- prettier --write {{changed}}",
     "prettier-watch:default": "onchange 'src/**/*.ts' -- prettier --write {{changed}}",
-    "test": "jest --config tools/jest.config.js --coverage -u",
```

Comment:
To update the snapshot if there is a change, it is necessary to have a flag -u. Otherwise, the snapshot test case would fail.

Reply:
Done